### ISSUE TITLE
use libremesh directly

### ISSUE THREAD

**Issue by nicopace** [luandro/meshpi#3]
LibreMesh is compatible (AFAIK) with Raspberry Pi, so you can run an image directly there.
Any reason why not to do it that way?

**Comment by luandro** [comment]
Good point. Since I've decided to use routers with Libre Mesh instead of Pis, I've given up on making this work.

I'll gladly take a PR in case someone else wants to give this a try.

**Comment by fabiengagne** [comment]
Hey nicopace, you've made the LibreMesh image work? I've tried the image from LibreMesh Chef's imagebuilder to my Pi Zero W, bare with only power supply (as my RT5370 haven't arrived yet). The LED blinks a few times then nothing. Could you point me to a link as to how to connect ssh or otherwise?

luando I will likely try yours too.

**Comment by luandro** [comment]
I haven given up to make it work for now @fabiengagne.

**Comment by nicopace** [comment]
@fabiengagne I would recommend to install it on a router and connect the Pi to the router.
Routers are much better at routing, and Pis at applications :)

**Comment by luandro** [comment]
That's the conclusion I reached also, and what I'll be using for the mesh setup.
